dallinger/recruiters.py

```diff
@@ -689,6 +857,16 @@ class MTurkRecruiterException(Exception):
 mturk_routes = flask.Blueprint("mturk_recruiter", __name__)
 
 
+def _mturk_service_from_config(sandbox):
```

Comment:
Doesn't this already exist in dallinger.command_line?

Reply:
Actually I see that this was already defined in recruiters.py so you haven't really changed anything. NVM